node/network/pov-distribution/src/lib.rs

```diff
@@ -557,45 +656,58 @@ impl PoVDistribution {
 	async fn run(
 		self,
 		mut ctx: impl SubsystemContext<Message = PoVDistributionMessage>,
-	) -> SubsystemResult<()> {
-		let mut state = State {
-			relay_parent_state: HashMap::new(),
-			peer_state: HashMap::new(),
-			our_view: View(Vec::new()),
-			metrics: self.metrics,
-		};
+	) -> error::Result<()> {
```

Comment:
I suggest to revert the return type back to `SubsystemResult<()>` or even remove it.
The errors should not propagate.